### PR TITLE
Since Django 1.9 deprecated 'django.db.models.loading', replace get_model by django.apps.apps.get_model

### PR DESCRIPTION
**Pull request by gogobook**
Replace get_model by AppConfig.get_model. Upgrade requirements.txt. Let Spirit can run by Django 1.9. But it can't pass test.

### CODE REVIEW

**Comment by henri-hulski** [comment]
I think .DS_Store could be removed from the PR.

File: requirements.txt [hunk 1]
```diff
@@ -1,9 +1,9 @@
-Django>=1.8,<1.9
+Django>=1.9,<1.10
```
Comment (nitely) [inline requirements.txt]: This should be `Django>=1.8,<1.10`, DJnago 1.8 support won't be dropped.

File: spirit/comment/poll/managers.py [hunk 2]
```diff
@@ -3,7 +3,7 @@
 from __future__ import unicode_literals
 
 from django.db import models
-from django.db.models.loading import get_model
+from django.apps.config import detached AppConfig
```
Comment (nitely) [inline spirit/comment/poll/managers.py]: This should be `from django.apps import apps` then `apps.get_model(...)`

btw, what is `detached`?? seems like some obscure Python 3 feature, it raises a syntax error in python 2. I'm very curious about this.

**Comment by nitely** [comment]
> I think .DS_Store could be removed from the PR.

:+1: 
You could even add it to gitignore.

Notice this PR is blocked by #91 and travis falling.

**Comment by nitely** [comment]
superseded by #137

**Pull request closed without merging**